docs/ddl-of-carbondata.md

```diff
@@ -234,7 +234,8 @@ CarbonData DDL statements are documented here,which includes:
       * TIMESTAMP
       * DATE
       * BOOLEAN
-   
+	  * FLOAT
```

Comment:
indentation is wrong, change the settings to replace tabs with spaces in your editor